RestaurantReservation.DB/Exceptions/RecordDoesNotExistException.cs

```diff
@@ -0,0 +1,10 @@
+namespace RestaurantReservation.DB.Exceptions;
+
+public class RecordDoesNotExistException : InvalidOperationException
+{
+    public RecordDoesNotExistException() {  }
```

Comment:
Its more proper to name it entity instead of record, change the other exception also